README.md

```diff
@@ -4,6 +4,113 @@
 
 This library provides instrumentation for CDI modules in Java EE applications.
 
+## Usage
+
+For a concrete and comprehensive set of examples, refer to the `opentracing-cdi-example`
+module.
+
+### Injecting a tracer
+
+In your managed beans, you can get an instance of the appropriate tracer via:
+
+```java
+@Inject
+Tracer tracer;
+```
+
+If you have your own `Tracer` producer, you'll need to mark your own as an
+`@Alternative`
```

Comment:
The options, as I see it, are:

- Have a non-qualified producer for `Tracer`, requiring developers to either qualify their own tracer or specify an alternative via `beans.xml`
- Have a qualified producer for `Tracer`, requiring developers to use this qualifier every time they want to use our producer.

I believe that our producer will be the most commonly used, so, the easiest for most developers will be to just have `@Inject Tracer tracer` and get a Tracer as returned by `GlobalTracer.get()`.

I'm preparing some tests, so that it's clear what are the options.